spec/services/proforma_service/convert_task_to_exercise_spec.rb

```diff
@@ -324,7 +345,6 @@
           id: exercise.id,
           title: task.title,
           description: task.description,
-          instructions: task.meta_data[:CodeOcean][:instructions],
```

Comment:
We removed instructions from CodeHarbor, right? Because CodeOcean still has them.

Reply:
As the `instructions` are no longer in use, dropping it here is fine.